Guspini

eurru opened the pull request:
Reinserito i dati relativi al comune di Guspini

eurru committed:
Inserito Guspini

Inserito Guspini. Per ora il periodo è da ottobre a dicembre 2014 e gennaio 2015

eurru committed:
Guspini 2014

Calendario 2014. Per ora il periodo inserito riguarda i mesi da ottobre al dicembre.
Da inserire i mesi precedenti

eurru committed:
Guspini 2015

Inserito il mese di gennaio 2015. In attesa del nuovo calendario

eurru committed:
Aggiornamento

Sostituito la voce "vetro - alluminio" con "vetro e alluminio"

eurru committed:
Aggiornamento Guspini 2015

Sostituito "vetro - alluminio" con "vetro e alluminio"

eurru committed:
Aggiornamento Guspini 2014

Aggiunti i dati da febbraio a settembre

eurru committed:
Aggiornamento

Modificato arco temporale per l'anno 2014

eurru committed:
Update guspini.schedule.2014.json

trumbitta commented on data/guspini.schedule.2014.json:
```diff
@@ -0,0 +1,5448 @@
+ "city": "Guspini",
+ "year": 2014,
+ "schedule": [
+	//febbraio
```
I commenti in JSON non sono ammessi. Dovresti rimuoverli tutti.
Puoi verificare se il JSON ha errori, con un linter. Ad esempio: http://jsonlint.com/

Credo di averti fuorviato io inserendone qualcuno nella guida a scopo esplicativo... dovrò trovare un modo migliore di esprimere gli stessi concetti!

trumbitta commented on data/guspini.schedule.2014.json:
```diff
@@ -0,0 +1,5448 @@
+ "schedule": [
+	//febbraio
+	{
+     "month": 02,
```
I numeri relativi al "month" devono essere interi, per cui `02` non è accettato da JSON (e da Aliga :smile:) mentre la forma corretta è `2`.

Questo è in qualche modo spiegato nella guida, ma mi rendo conto che non lo è in modo inequivocabile. Un'altra parte che dovrò capire come esprimere meglio!

trumbitta commented on data/guspini.years.json:
```diff
@@ -0,0 +1,15 @@
+    },
+    {
+      "year": 2015,
+      "validfrom": "2015-01-01"
```
Manca una vigola! :smile_cat:

eurru committed:
Corretto anno 2015

eurru committed:
Aggiunto Guspini

trumbitta commented:
Non modificare `cities.json`, quello lo faccio io una volta accettata la pull-request per evitare che si creino conflitti di modifica tra pull-request contemporanee.

eurru commented:
ok

From: William Ghelfi 
Sent: Monday, October 20, 2014 1:38 PM
To: aligaorg/aliga-data 
Cc: eurru 
Subject: Re: [aliga-data] Guspini (#7)

Non modificare cities.json, quello lo faccio io una volta accettata la pull-request per evitare che si creino conflitti di modifica tra pull-request contemporanee.

—
Reply to this email directly or view it on GitHub.

eurru committed:
Eliminati commenti e corretta numerazione month

Aggiornato il file con l'eliminazione dei commenti e della numerazione dei month (senza lo 0 iniziale)

eurru committed:
nuova correzione month

trumbitta commented:
@eurru potresti fare un nuovo commit su `cities.json` per cancellare l'aggiunta di Guspini? :smiley_cat: 
Se vedo bene, è l'unica cosa che manca prima dell'ingresso di Guspini in Aliga :shipit:

eurru committed:
eliminato guspini
